i18next/i18next.d.ts

```diff
@@ -87,6 +87,7 @@ interface I18nextStatic {
     };
     init(callback?: (t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;
     init(options?: I18nextOptions, callback?: (t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;
+    init(options?: I18nextOptions, callback?: (err: any, t: (key: string, options?: any) => string) => void ): JQueryDeferred<any>;

```

Comment:
L90 works with or without L89, but L89 does not work even without L90, at least for me. I left L89 just in case it works for other people, although it seems to be incorrect according to the implementation and documentation of i18n.init(). Should I delete it?